src/pymmcore_plus/metadata/serialize.py

```diff
@@ -36,6 +37,8 @@ def encode_hook(obj: Any, raises: bool = True) -> Any:
         return obj.item()
     if isinstance(obj, timedelta):
         return obj.total_seconds()
+    if isinstance(obj, Enum):
```

Comment:
@wl-stepp here is the other way this could be solved.  Since we *did* call `model_dump` above, there were now `Enums` in the value (as long as there was a grid plan in the `MDASequence`)... and msgspec knew how to handle enums but stdlib json.dumps doesn't.  So this fixes it as well